ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/forkchoice/ProtoNodeData.java

```diff
@@ -102,6 +118,7 @@ public String toString() {
         .add("stateRoot", stateRoot)
         .add("executionBlockHash", executionBlockHash)
         .add("isOptimistic", isOptimistic)
+        .add("checkpoints", checkpoints)
```

Comment:
Went the other way and changed equals/hashCode to use direct variable access since we don't normally use getters for internal access (IntelliJ defaulted to generating with getters I think).